pkg/pbflow/proto.go

```diff
@@ -90,10 +90,27 @@ func FlowToPB(fr *model.Record, s *ovnobserv.SampleDecoder) *Record {
 	}
 	pbflowRecord.DupList = make([]*DupMapEntry, 0)
 	for _, intf := range fr.Interfaces {
-		pbflowRecord.DupList = append(pbflowRecord.DupList, &DupMapEntry{
+		entry := DupMapEntry{
 			Interface: intf.Interface,
 			Direction: Direction(intf.Direction),
-		})
+			Udn:       "",
+		}
+		if s != nil {
```

Comment:
Doing this in protobuf decode will cause the same issue as what I mentioned here: https://github.com/netobserv/netobserv-ebpf-agent/pull/286#pullrequestreview-2230254996
.. and which made us having this bug: https://issues.redhat.com/browse/NETOBSERV-2009
I think it doesn't make sense to have any enrichment set in protobuf decode. There should be in a common path, not something that may end up not being invoked. IMO we should fix that in a first place rather than adding technical debt.

Reply:
I think adding new pipeline might be very risk at this stage given the unstability e have can this be deferred to 1.9 ?

Reply:
An option could be to create a new stage (like in [this area](https://github.com/netobserv/netobserv-ebpf-agent/blob/ca5b9445508a8befa8fe65eefd8f29d460733f79/pkg/agent/agent.go#L504-L515)), or perhaps inject some enrichment callback directly in `NewRecord` [like it's done for interfaceNamer](https://github.com/netobserv/netobserv-ebpf-agent/blob/main/pkg/model/record.go#L89), wdyt?

Reply:
We should fix NETOBSERV-2009 in 1.8 so if we fix it for events we could surely fix it as well for udns?

Reply:
we still in TP for network events so no urgency for CLI support I think for 1.8 I will try to explode the new stage option